### PR TITLE
Add GetMethod operation

### PR DESCRIPTION
**Pull request by JakeChampion**
This PR adds the `GetMethod` operation as defined in section [7.3.9](http://www.ecma-international.org/ecma-262/6.0/#sec-getmethod).

### CODE REVIEW

File: es6.js [hunk 1]
```diff
@@ -224,6 +224,18 @@ var ES6 = assign(assign({}, ES5), {
 	// https://people.mozilla.org/~jorendorff/es6-draft.html#sec-samevaluezero
 	SameValueZero: function SameValueZero(x, y) {
 		return (x === y) || ($isNaN(x) && $isNaN(y));
+	},
+
+	// ES6 7.3.9 - http://www.ecma-international.org/ecma-262/6.0/#sec-getmethod
+	GetMethod: function GetMethod(obj, p) {
```
Comment (ljharb) [inline es6.js]: this method needs to throw if `p` is not a property key (if we don't have `IsPropertyKey` defined already, we may want to add it)

File: es6.js [hunk 2]
```diff
@@ -224,6 +224,18 @@ var ES6 = assign(assign({}, ES5), {
...
+		var func = obj[p];
```
Comment (ljharb) [inline es6.js]: this lookup should be done with http://www.ecma-international.org/ecma-262/6.0/#sec-getv, and not directly with a property lookup

File: es6.js [hunk 3]
```diff
@@ -224,6 +224,18 @@ var ES6 = assign(assign({}, ES5), {
...
+		if (func === undefined || func === null) {
```
Comment (ljharb) [inline es6.js]: please use `typeof` to check undefined, since undefined can be redefined.

File: es6.js [hunk 4]
```diff
@@ -224,6 +224,18 @@ var ES6 = assign(assign({}, ES5), {
...
+			return undefined;
```
Comment (ljharb) [inline es6.js]: similarly, `return;` should be sufficient here

File: es6.js [hunk 5]
```diff
@@ -224,6 +224,18 @@ var ES6 = assign(assign({}, ES5), {
...
+		}
+		if (this.IsCallable(func)) {
+			return func;
+		}
+		throw new TypeError(p + 'is not a function');
```
Comment (ljharb) [inline es6.js]: i think this needs a space before "is"

**Comment by ljharb** [comment]
@JakeChampion now that #17 is merged, would you mind rebasing and/or checking "allow edits"? :-)

**Comment by JakeChampion** [comment]
@ljharb Checked the "Allow edits from maintainers" checkbox. I've also added `GetV`. I'm not sure what `ReturnIfAbrupt` means, it is in a lot of the spec, specifically it is in Step 3 of `GetV`.

**Comment by ljharb** [comment]
`ReturnIfAbrupt` basically means "if it threw, throw" - in JS we can skip over most of those.

**Pull request merged**